src/test/kotlin/com/github/avrokotlin/avro4k/stuff.kt

```diff
@@ -0,0 +1,26 @@
+package com.github.avrokotlin.avro4k
```

Comment:
Maybe this file is better named utils.kt?

Reply:
hahaha oops 😆 i'll do it